### PR TITLE
Create email-testing.md

### PR DESCRIPTION
**Pull request by davisshaver**

### CODE REVIEW

**Comment by davisshaver** [comment]
@mpatek @danielbachhuber review?

**Comment by danielbachhuber** [comment]
Could we put this in a `tools` directory? Or another suggestion?

The doc doesn't seem very descriptive...

**Comment by davisshaver** [comment]
@danielbachhuber length & quality aren't linked... this describes the email testing workflow as we've documented so far. 

Don't care about directory, it should be easily changeable later anyway. Think the best part of these repos is keeping them evergreen so trying to overthink any particular decision.

**Comment by danielbachhuber** [comment]
> Don't care about directory, it should be easily changeable later anyway.

Changing directory breaks all previous links to the document, so we should give it at least a little bit of upfront thought. How about `development`?

**Comment by davisshaver** [comment]
@danielbachhuber Yeah sorry you right. I like `/tools/` and then we can rename this one to be Litmus, since it's so specific.

File: email/email-testing.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+1. Login to Litmus using credentials in Meldium.
+1. Hit the `New Email Test` button.
```
Comment (danielbachhuber) [inline email/email-testing.md]: @davisshaver How do you get your email sample into Litmus?

File: email/email-testing.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+1. Choose your clients.
```
Comment (danielbachhuber) [inline email/email-testing.md]: Do we have good data on what our clients are?

**Comment by davisshaver** [comment]
@danielbachhuber - review/merge?

**Pull request merged**